.github/workflows/ci-tests-fabric.yml

```diff
@@ -137,9 +137,8 @@ jobs:
       working-directory: tests/tests_fabric
       # NOTE: do not include coverage report here, see: https://github.com/nedbat/coveragepy/issues/1003
       run: |
-        echo $GITHUB_RUN_ID
         python -m coverage run --source ${{ env.COVERAGE_SCOPE }} \
-          -m pytest -v --timeout=30 --durations=50 --random-order-seed=$GITHUB_RUN_ID
```

Comment:
You could just hardcode 42 here without reverting the entire PR. PyTorch tests are working and don't need the revert